fairscale/utils/testing.py

```diff
@@ -116,6 +117,30 @@ def torch_version() -> Tuple[int, ...]:
     return tuple(int(n) for n in numbering)
 
 
+_smi_ver = None
+
+
+def torch_cuda_version(compiled: bool = False) -> Tuple[int, ...]:
```

Comment:
added a new function to either get torch compiled cuda version or Nvidia-smi reported cuda version.